sql/core/src/test/resources/sql-tests/inputs/postgreSQL/join.sql

```diff
@@ -743,20 +743,20 @@ select * from a left join b on i = x and i = y and x = i;
 --
 -- test NULL behavior of whole-row Vars, per bug #5025
 --
-select t1.q2, count(t2.*)
+select t1.q2, count(t2.q1, t2.q2)
```

Comment:
nit: Could you leave some comments here about why we updated the tests?

Reply:
fixed